pkg/proxy/apis/kubeproxyconfig/types.go

```diff
@@ -168,6 +177,14 @@ const (
 	ProxyModeKernelspace ProxyMode = "kernelspace"
 )
 
+type InterfaceType string
+
+const (
+	DefaultRoute InterfaceType = "default-route"
+	IPv4ZeroCIDR InterfaceType = "0.0.0.0/0"
```

Comment:
everywhere you use a default, it should be both IPV4 and IPV6.

Reply:
Changed to "all-interfaces" now.